### PR TITLE
Search Page small things

### PR DESCRIPTION
**Pull request by k-atej**
Styling/minor fixes to the search page
- added descending sorting to genus & species
- set species sort to default
- styled table to be more readable
- added ability to clear filters 
- added column of checkboxes (not currently functional, but representative of "selecting" a plant)

*does not fix pagination/sort conflict bug

### CODE REVIEW

**Comment by k-atej** [comment]
Fixed the Category/Search parameter conflict with pagination, as well as sort order conflict with pagination.

**Comment by reidst** [comment]
There's still an issue sorting when trying to go from `genus_desc` to `genus`, but that can be fixed by changing Pages/Search/Index.cshtml.cs line 67 like this:
```diff
  public async Task OnGetAsync(...)
  {
-     GenusSort = String.IsNullOrEmpty(sortOrder)? "genus": "genus_desc";
+     GenusSort = (String.IsNullOrEmpty(sortOrder) || sortOrder.Equals("genus_desc"))? "genus": "genus_desc";
      SpeciesSort = String.IsNullOrEmpty(sortOrder)? "species_desc": "";
```

File: Pages/Search/Index.cshtml.cs [hunk 1]
```diff
@@ -53,32 +53,54 @@ public IndexModel(csci340_iseegreen.Data.ISeeGreenContext context, IConfiguratio
...
+        public string CurrentCat {get; set;}
 
 
-        public async Task OnGetAsync(string sortOrder, string SearchString, int? pageIndex)
+
+        public async Task OnGetAsync(string sortOrder, string FamilyString, string GenusString, string SearchString, string? CategoryFilter, int? pageIndex)
         {
-            SpeciesSort = String.IsNullOrEmpty(sortOrder) ? "species": "";
-            GenusSort = String.IsNullOrEmpty(sortOrder) ? "genus": "";
+            GenusSort = String.IsNullOrEmpty(sortOrder)? "genus": "genus_desc";
```
Comment (reidst) [inline Pages/Search/Index.cshtml.cs:67]: Line 67 sort fix:
```csharp
GenusSort = (String.IsNullOrEmpty(sortOrder) || sortOrder.Equals("genus_desc"))? "genus": "genus_desc";
```

**Review by reidst — CHANGES_REQUESTED** [review]

File: Pages/Search/Index.cshtml.cs (hunk 1 above)
Comment (k-atej) [reply Pages/Search/Index.cshtml.cs]: Done!

**Review by reidst — APPROVED** [review]

**Pull request merged**